gateway-service-knoxssout/pom.xml

```diff
@@ -68,5 +68,9 @@
             <artifactId>gateway-test-utils</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.apache.knox</groupId>
```

Comment:
Which class is references from gateway-server from gateway-service-knoxssout?

Reply:
The ConcurrentSessionVerifier is in gateway-server.

Reply:
We can solve this by inverting the dependencies. Let's introduce an interface fo the ConcurrentSessionVerifier and put it under gateway-spi. In fact the ConcurrentSessionVerifier is already a good interface name, I would rename the implementation to something like InMemoryConcurrentSessionVerifier. So the implementation can stay in gateway-server and the interface in gateway-spi. WebSSOutResource and WebSSOResource only needs to know the interface and not the implementation. After that we can delete this dependency.

Reply:
Thats a good idea thank you, i implemented it.

Reply:
Cool, thanks.